assets/js/theme/common/product-card.js

```diff
@@ -0,0 +1,30 @@
+const focusedCardClass = 'focused-card';
+const focusableElementsSelector = '[href], button, input, textarea, select, details, [tabindex]';
+const previousCard = {
```

Comment:
I can see only usage of "$previous" link. Why do we need object in this case?

Reply:
if we will use simple variable - we will scope its value, but we need to scope the link to the value for repeat usage

Reply:
It does not depend or what we are using, it depends on data type that we write into variable

Reply:
  yes, that is why I added object as variable value

Reply:
It is unnecessary as you write jquery object in a $previous var

Reply:
at the beginning I have no $previous element. Of course I can use {} instead of null, but null easier to check so I decided to do it in react useRef style:) 
but anyway now it is removed - polyfil is realy better idea. thanks)